.github/workflows/reusable-secrets-analysis.yml

```diff
@@ -2,25 +2,14 @@ on:
   workflow_call
 
 jobs:
-  trufflehog:
+  trufflehog:  
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
         with:
           fetch-depth: 0
 
-      - uses: actions/setup-python@v4
+      - name: Secret Scanning
+        uses: trufflesecurity/trufflehog@v3.63.8
         with:
-          python-version: 3.x
-
-      - name: Pip install trufflehog
-        shell: bash
-        run: |
-          python -m pip install trufflehog
-
-      - name: Scan for secrets with trufflehog
-        shell: bash
-        run: |
-          export LAST_TAG_HASH=$(git rev-list -1 $(git describe --abbrev=0))
-          trufflehog --regex --entropy True --since_commit "${LAST_TAG_HASH}" \
-              --exclude_paths .trufflehog.txt file://"${PWD}"
+          extra_args: --only-verified --exclude_paths .trufflehog.txt
```

Comment:
It looks like `--exclude_paths .trufflehog.txt` doesn't work right now due to: https://github.com/trufflesecurity/trufflehog/issues/2248

I'd actually be tempted to stop supporting this argument in our actions, and add it back later if needed.

Reply:
@jtherrmann as I see it, there's three options to move forward, which I've listed in order of *my* preference (1 being most preferable):
1. drop supporting `.truffhog.txt`  and move forward without `--exclude_paths`
2. install trufflhog ourselves and use it's cli instead of the action
3. roll back to a version of this action that does support `--exclude_paths` 


I think (1) is okay, because this PR would drop the entropy check (the most common false positive for us) and switch over to the secret verification API so we *may* not need any of our current excludes anyways. Also, if we did want to add excludes, it'd be better to exclude *globs* instead of *files*, which we could do with the *new* `--exclude-globs` option:
```
  -x, --exclude-paths=EXCLUDE-PATHS
                                 Path to file with newline separated regexes for files to exclude in scan.
      --exclude-globs=EXCLUDE-GLOBS
                                 Comma separated list of globs to exclude in scan. This option filters at the `git log` level, resulting in faster scans.
```
This would force us to re-evaluate all of our excludes and cut over to globs if needed, but could pose more work to start using this action.

while (2) is find and would work, but we'd just be re-creating the GitHub action they now provide and maintain for us.

And (3) is probably not a good idea because it might not work and doesn't include new secrets/fixes/etc.

Reply:
The work I was going to do to move this out of draft was:
* try the new action on a known false positive and see how it works
* remove the excludes from a repo and see what it does

Reply:
Probably should add:
* try the new action on a true positive and confirm that it fails

Reply:
Verified the second bullet point at https://github.com/ASFHyP3/hyp3-docs/pull/406

Also verified the first and third bullet points (see my comment below).